Library/Formula/sfml.rb

```diff
@@ -27,9 +17,44 @@ class Sfml < Formula
   depends_on "glew"
   depends_on "jpeg"
   depends_on "libsndfile"
+  depends_on "doxygen" => :optional
+
+  # Patch the bundled libs wording that causes linking errors. Accepted upstream.
+  # https://github.com/Homebrew/homebrew/pull/35279
+  # https://github.com/ludvichzme/SFML/commit/c3f4f62f306f6f9b979b8c0cb2bfb3968712c8fa
+  patch :DATA
```

Comment:
This patch should be applied to the source archive for SFML 2.2 only. (Maybe that's not the case right now?) We will provide it upstream before the next release (2.3) so it should not be needed afterwards.